src/uct/base/uct_iface.c

```diff
@@ -619,6 +619,12 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
     return ep->iface->ops.ep_connect_to_ep(ep, dev_addr, ep_addr);
 }
 
+ucs_status_t uct_ep_connect_to_ep_v2(uct_ep_h ep,
+                                     const uct_ep_connect_to_ep_params_t *params)
+{
+    return ep->iface->ops.ep_connect_to_ep_v2(ep, params);
```

Comment:
maybe return `UCS_ERR_NOT_IMPLEMENTED` for now?

Reply:
as discussed offline - non needed